### PR TITLE
Fix wrong C++ link on Ruby documentation

### PR DESCRIPTION
**Pull request by vnbrs**

### CODE REVIEW

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @vnbrs isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Comment by vnbrs** [comment]
> Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.
> 
> After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.
> 
> * User @vnbrs isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf
> 
> _Regards, CLA GitHub bot_

:shipit:

**Comment by jtattermusch** [comment]
@vnbrs  please sign the CLA, otherwise we can't merge.

**Comment by vnbrs** [comment]
I have already signed it

**Comment by vnbrs** [comment]
> @vnbrs please sign the CLA, otherwise we can't merge.

I have already signed it

**Comment by vnbrs** [comment]
> Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.
> 
> After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.
> 
> * User @vnbrs isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf
> 
> _Regards, CLA GitHub bot_

I signed it

**Comment by vnbrs** [comment]
> Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.
> 
> After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.
> 
> * User @vnbrs isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf
> 
> _Regards, CLA GitHub bot_

I signed it

File: docs/tutorials/basic/ruby.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ a special gRPC Ruby plugin.
 
 If you want to run this yourself, make sure you've installed protoc and followed
 the gRPC Ruby plugin [installation
-instructions](https://github.com/grpc/grpc/blob/{{ site.data.config.grpc_release_tag }}/src/cpp/README.md#make) first):
+instructions](https://github.com/grpc/grpc/blob/{{ site.data.config.grpc_release_tag }}/src/ruby/README.md) first):
```
Comment (jtattermusch) [inline docs/tutorials/basic/ruby.md]: Acually, even this looks likes it must be a type, this part is about installing protoc and grpc_ruby_plugin for protoc and https://github.com/grpc/grpc/blob/master/src/ruby/README.md doesn't mention how that's supposed to be done for ruby.

@apolcyn  should the README.md (and/or this document) mention the usage of "grpc-tools" package? https://rubygems.org/gems/grpc-tools/versions/1.18.0

Comment (apolcyn) [reply docs/tutorials/basic/ruby.md]: yeah, you're right. I'll merge this and open a follow-up

Comment (apolcyn) [reply docs/tutorials/basic/ruby.md]: actually, I don't have write-access for this repo, so I can't merge

**Pull request merged**